src/samples/audio/smart_amp_llext/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ SET_PROPERTY(GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS TRUE)
 set(MODULE "smart_amp_test")
 cmake_path(SET SOF_BASE NORMALIZE ${PROJECT_SOURCE_DIR}/../../../..)
 
+file(STRINGS ${CMAKE_CURRENT_LIST_DIR}/../${MODULE}.toml uuids REGEX "^[ \t]*uuid *=")
+
+file(WRITE ${PROJECT_BINARY_DIR}/llext.uuid "")
+
+foreach(line IN LISTS uuids)
+	string(REGEX REPLACE "^[ \t]*uuid *= \"([0-9A-F\-]*)\"" "\\1" uuid ${line})
+	file(APPEND ${PROJECT_BINARY_DIR}/llext.uuid "${uuid}\n")
+endforeach()
```

Comment:
why is this a smart_amp specific operation? Are we going to replicate this for each and every module?

Reply:
@plbossart so far this CMakeLists.txt would be largely repeated for each new module, yes. But we plan to switch to using recently added Zephyr cmake support for LLEXT code ASAP